__tests__/components/organisms/ConversationWindow.spec.js

```diff
@@ -0,0 +1,60 @@
+import { shallowMount } from "@vue/test-utils";
+import ConversationWindow from "../../../src/components/organisms/ConversationWindow";
+import TextInput from "../../../src/components/atoms/TextInput";
+import MessageHeader from "../../../src/components/atoms/MessageHeader";
+import { createStore, useStore } from "vuex";
+import { i18n } from "../../../i18n";
+import chatMessages from "../../../src/store/modules/chatMessages";
+jest.mock("../../../src/store/modules/chatMessages");
+import emails from "../../../src/store/modules/emails";
+jest.mock("../../../src/store/modules/emails");
+import inbox from "../../../src/store/modules/inbox";
+jest.mock("../../../src/store/modules/inbox");
+
+describe("ConversationWindow component", () => {
+  let wrapper;
+  let store;
+  beforeAll(() => {
+    chatMessages.actions.sendChatMessage = jest.fn();
+    store = createStore({
+      modules: {
+        chatMessages,
+        emails,
+        inbox,
+      },
+    });
+
+    wrapper = shallowMount(ConversationWindow, {
+      global: {
+        provide: {
+          store,
+        },
+        plugins: [i18n],
+      },
+    });
+  });
+
+  it("Check for message header component", () => {
+    expect(wrapper.findComponent(MessageHeader).exists()).toBe(true);
+  });
+
+  it("Check for text input component", () => {
+    expect(wrapper.findComponent(TextInput).exists()).toBe(true);
+  });
+
+  it("Test sendChatMessage", async () => {
+    // const textInput = wrapper.find('#textInputInput')
+    // await textInput.setValue('test value')
+    // const sendTextInputButton = wrapper.find("#sendTextInputButton");
+    // sendTextInputButton.trigger('click');
```

Comment:
Should we keep these comments?

Reply:
I wanted to keep as an example but I've already done in another PR I've removed them